include/deal.II/lac/affine_constraints.templates.h

```diff
@@ -1924,12 +1924,7 @@ namespace internal
       const int n_blocks = 1 + (n_constraints - 1) / CUDAWrappers::block_size;
       set_zero_kernel<<<n_blocks, CUDAWrappers::block_size>>>(
         constrained_local_dofs_device, n_constraints, vec.get_values());
-#  ifdef DEBUG
-      // Check that the kernel was launched correctly
-      AssertCuda(cudaGetLastError());
-      // Check that there was no problem during the execution of the kernel
-      AssertCuda(cudaDeviceSynchronize());
-#  endif
+      AssertCudaKernel;
```

Comment:
Hmm... I would prefer this to look like a function call to be similar to the other asserts.
```suggestion
      AssertCudaKernel();
```

Reply:
Yes, I strongly favor this as well.

Reply:
done